app/build.gradle

```diff
@@ -836,3 +836,6 @@ android.applicationVariants.all { variant ->
         dependsOn "package${variant.name.capitalize()}"
     }
 }
+
+// Enable expiration by major version.
+ext.gleanExpireByVersion = Config.androidComponentsMajorVersionName()
```

Comment:
Not sure if this is the best way to get the major version.  It is the simplest way though.

Reply:
@rocketsroger I think we can add a method specifically for `Fenix` that will work with both the Nightly and Release versioning schemes. Happy to help with that.